CHANGELOG.md

```diff
@@ -95,6 +96,7 @@
  3. Design changes:
  - Split features into API modules for each hardware variant
  - Fixed/updated/added CLI commands
+ 4. HPE Synergy Image Streamer Support:
```

Comment:
@jsmartt , thinking about it, this version does not yet support Image Streamer, it's just the architecture for future support.

I think we should just add "Design Changes" in the new section above. What do you think about this?